recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "cptec-model" %}
-{% set version = "0.1.8" %}
+{% set version = "0.1.9" %}
```

Comment:
The fact that this is passing with the same checksum as 0.1.8 s very odd! Also, the bot should pick up a new version, so maybe something else is amiss here.